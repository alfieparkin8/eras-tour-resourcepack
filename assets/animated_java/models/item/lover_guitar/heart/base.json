{"parent":"animated_java:item/lover_guitar/base","textures":{"2":"minecraft:block/outfits/lover/heart_layer_1"}}