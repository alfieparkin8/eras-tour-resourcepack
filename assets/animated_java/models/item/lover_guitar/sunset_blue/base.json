{"parent":"animated_java:item/lover_guitar/base","textures":{"1":"minecraft:block/guitars/lover_blue"}}