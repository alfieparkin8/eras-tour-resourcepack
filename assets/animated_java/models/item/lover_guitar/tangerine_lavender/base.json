{
	"parent": "animated_java:item/lover_guitar/base",
	"textures": {
		"1": "animated_java:item/lover_guitar/tangerine_layer_1",
		"2": "animated_java:item/lover_guitar/lover_lavender",
		"5": "animated_java:item/lover_guitar/tangerine_boots_layer_1"
	}
}