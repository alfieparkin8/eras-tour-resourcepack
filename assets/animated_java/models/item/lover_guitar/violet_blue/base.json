{"parent":"animated_java:item/lover_guitar/base","textures":{"1":"minecraft:block/guitars/lover_blue","2":"minecraft:block/outfits/lover/violet_layer_1"}}