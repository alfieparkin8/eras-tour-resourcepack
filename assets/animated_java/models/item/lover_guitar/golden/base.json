{"parent":"animated_java:item/lover_guitar/base","textures":{"2":"minecraft:block/outfits/lover/golden_layer_1"}}